package.json

```diff
@@ -67,16 +67,17 @@
     "redux-devtools": "3.4.0",
     "redux-devtools-dock-monitor": "1.1.2",
     "redux-devtools-log-monitor": "1.3.0",
-    "redux-immutable-state-invariant": "1.2.3",
+    "redux-immutable-state-invariant": "2.1.0",
```

Comment:
i have found a bug when you run the app with debug=true
there is a problem here
```
var immutable = __webpack_require__(/*! redux-immutable-state-invariant */ "./node_modules/redux-immutable-state-invariant/dist/index.js")();
```
![image](https://user-images.githubusercontent.com/11991428/64680018-2fd50180-d47d-11e9-9a9b-e7af65c8e50a.png)